### DISCUSSION TITLE
Refactor of the code and x-installation fix

### DISCUSSION THREAD

**Discussion by guerrerotook** [guerrerotook/securitas-direct-new-api#192] [Announcements]
Hello everyone, this release is thanks to @cantupaz that mostly wrote the refactoring of the code and the fix for the x-installation issue #179  that we have been experiencing. 

Please provide feedback of this release.

<hr /><em>This discussion was created from the release <a href='https://github.com/guerrerotook/securitas-direct-new-api/releases/tag/v2.6.0.0'>Refactor of the code and x-installation fix</a>.</em>

**Comment by lukelalo** [comment]
Thanks for the good work! 👏 👏 👏

**Comment by joandomenech** [comment]
Now it works perfectly. Thank you for hard work 💪
